ArduPlane/radio.cpp

```diff
@@ -118,9 +118,14 @@ void Plane::init_rc_out_aux()
 */
 void Plane::rudder_arm_disarm_check()
 {
+    const int16_t rudder_in = channel_rudder->get_control_in();
+    if (rudder_in == 0) {
```

Comment:
Is ==0 too strict?

Reply:
no, == 0 is exactly right. It is an integer, and == 0 means it is within the dead-zone. Any non-zero value means the rudder is not centered